packages/vision/src/index.js

```diff
@@ -1404,6 +1737,22 @@ Vision.findImages_ = function(images, callback) {
 };
 
 /**
+ * Format a raw crop hint annotation response from the API.
+ *
+ * @private
+ */
+Vision.formatCropHintsAnnotation_ = function(cropHintsAnnotation, options) {
```

Comment:
Considering all the requests to remove verbose mode in other APIs, I'm a little hesitant about this. Maybe instead of completely altering the response we should just augment it with convenience properties.